shared/src/commonTest/kotlin/co/nimblehq/ic/kmm/suv/data/repository/UserRepositoryTest.kt

```diff
@@ -0,0 +1,77 @@
+package co.nimblehq.ic.kmm.suv.data.repository
+
+import co.nimblehq.ic.kmm.suv.data.remote.datasource.TokenRemoteDataSource
+import co.nimblehq.ic.kmm.suv.data.remote.model.TokenApiModel
+import co.nimblehq.ic.kmm.suv.data.remote.model.toToken
+import co.nimblehq.ic.kmm.suv.domain.repository.TokenRepository
+import io.kotest.matchers.shouldBe
+import io.mockative.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.runTest
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+@ExperimentalCoroutinesApi
+class UserRepositoryTest {
```

Comment:
I think you forgot to rename this test file 😂

Reply:
Nice catch :+1:

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/87/commits/2ecd741e68bacf7d8eb48432ec70c05a8bdda708